src/lib/utils/thorchain/lp.ts

```diff
@@ -147,6 +148,94 @@ export const getVolume = async (
   return fromThorBaseUnit(volume).times(runePrice).toFixed()
 }
 
+export const get24hSwapChangePercentage = async (
+  assetId: AssetId,
+  runePrice: string,
+  assetPrice: string,
+): Promise<{ volumeChangePercentage: number; feeChangePercentage: number } | null> => {
+  const poolAssetId = assetIdToPoolAssetId({ assetId })
+  const now = Math.floor(Date.now() / 1000)
+  const twentyFourHoursAgo = now - 24 * 60 * 60
+  const fortyEightHoursAgo = now - 2 * 24 * 60 * 60
+
+  const { data: current24hData } = await axios.get<MidgardSwapHistoryResponse>(
```

Comment:
Technically, we're already displaying the current, although not with the same `from=&to=` which may explain some of the discrepancies here - will revisit this as a direct follow-up, not blocking this PR so we can keep things moving

Reply:
Similar thought here - if we use the `interval` and `count` query strings we can end up with really weird data.

I think the `from` and `to` query strings will yield us the best results, or so I concluded from messing with these endpoints all day.